pages.id/linux/pacman-query.md

```diff
@@ -0,0 +1,36 @@
+# pacman --query
+
+> Kegunaan manajer paket Arch Linux.
+> Informasi lebih lanjut: <https://man.archlinux.org/man/pacman.8>.
+
+- Daftar paket yang diinstal beserta versinya:
```

Comment:
```suggestion
- Tampilkan daftar paket yang diinstal beserta versinya:
```

Because the original English version is meant to display a list, not to register a new package.

Reply:
(And for other examples starting with `daftar`